ceaos/actuators/actuator_commands.py

```diff
@@ -0,0 +1,20 @@
+from ..objects import Farm, Environment, Bed
```

Comment:
Is this file just to provide stubs for the rest of the code? Like fake actuators to use? Why not use artificial actuators here instead?